webpack/webpack.dev.config.js

```diff
@@ -109,6 +109,7 @@ module.exports = Merge.smart(commonConfig, {
       BASE_URL: 'localhost:1995',
       LMS_BASE_URL: 'http://localhost:18000',
       CREDENTIALS_BASE_URL: 'http://localhost:18150',
+      ECOMMERCE_BASE_URL: 'http://localhost:18130',
```

Comment:
Why add this if we aren't going to use it here?

Reply:
Considered that - I added it under the assumption that eventually this config setup will be coming from a common place and that it'd be convenient if they were all the same when that DRYing happened.